### PR TITLE
Changes From EGM 20/03/24

### PR DESCRIPTION
**Pull request by Jacrac04**
## Changes:
- Update Committee Roles - 16/0/4 (for/against/abstained)
- Fix Spelling Mistake - 14/2/4 (for/against/abstained)
- Remove Vote of No Confidence  - 20/0/0 (for/against/abstained)

Approved at EGM during MMCTF on the 20/03/24.

### CODE REVIEW

**Comment by Jacrac04** [comment]
Wait for SUSU approval before Merge

**Review by matthew-botten — APPROVED** [review]
Looks good, and susu has approved

**Pull request merged**
